src/Pipeline.h

```diff
@@ -90,6 +90,10 @@ struct Outputs {
     }
 };
 
+struct JITExtern;
+
+typedef std::map<std::string, JITExtern> NamedJITExterns;
```

Comment:
I'd mildly prefer not having this typedef, and making it more explicitly a map in the Pipeline API.